### DISCUSSION TITLE
Build Documentation is failing

### DISCUSSION THREAD

**Discussion by Alexminator99** [pointfreeco/swift-composable-architecture#3260] [Q&A] [answered]
Composable Architecture: 1.12.1
Xcode: 15.3
Swift: 5.10

I'm encountering the following error when attempting to build the documentation:

`error: Symbol with identifier 's:6Test10AppFeatureV4PathO5StateO' couldn't be found (in target 'Test' from project 'Test')`

I discovered an [old issue](https://github.com/swiftlang/swift/issues/60771) that appears related, but it doesn't directly address something I can resolve. Instead, it seems to be an issue with Composable Architecture itself.

If anyone has encountered this issue before, guidance on how to fix or work around it would be greatly appreciated.

**Comment by mbrandonw** [comment, marked as answer]
Hi @Alexminator99, this sounds related to this https://github.com/pointfreeco/swift-composable-architecture/discussions/2931, and as far as we can tell it is an Xcode issue.

**Reply by Alexminator99** [reply]
Thanks for sharing that thread. I will keep and eye on it and close this one as the question is repeated.
